### DISCUSSION TITLE
float16 computation・・・

### DISCUSSION THREAD

**Discussion by kamekame0202** [openai/whisper#1732] [Q&A]
Hi

I'm a beginner in programming.
I built it on Windows as per README.md.

When using Quadro M5000 GPU, the following error occurs.
"Error transcribing file on line Requested float16 compute type, but the target device or backend do not support efficient float16 computation."

Is there a way to use it with float16?

**Comment by phineas-pta** [comment]
your gpu doesnt support float16

**Reply by kamekame0202** [reply]
thank you.
I understand M5000 is not supported.

Is it possible to make the code work with compute_type to "float32" or "int8"?

**Reply by kamekame0202** [reply]
so sorry, I overlooked that there was a selection screen for float16 and float32.
I selected float32 and it worked.

**Comment by kamekame0202** [comment]
so sorry, I overlooked that there was a selection screen for float16 and float32.
I selected float32 and it worked.

thank you.
